gulpfile.js

```diff
@@ -69,10 +69,21 @@ function doLint(paths, exit) {
 gulp.task('default', ['clean', 'lint', 'build']);
 gulp.task('release', ['clean', 'lint', 'build', 'minimize']);
 
+const browserSync = require('browser-sync').create();
 gulp.task('watch', ['clean'], function () {
+    browserSync.init({
+        server: {
+            baseDir: './'
+        },
+        port: 8000,
+        open: false
+    });
+    require('opn')('http://localhost:8000/demo/');
+
     let gulpWatcher = gulp.watch(['gulpfile.js', 'src/**/*.js']);
 
-    gulpWatcher.on('change', function (e) {
+    gulpWatcher.on('change', function (e, ...args) {
```

Comment:
The syntax of `...args` is only supported on newer node.js versions
Since gulpfile.js will not be passed the Babel compiler, advanced syntax will break some old-version users like me :(
use `browserSync.reload.apply(browserSync, arguments);` instead please